### ISSUE TITLE
[Designer] Pic2Card dialog unusable at high zoom

### ISSUE THREAD

**Issue by paulcam206** [microsoft/AdaptiveCards#6312]
### Problem Description

Splitting this issue out of #6263...

WCAG 2.1 accessibility requirements state that UI should be [usable at 1280x768 at 400% zoom](https://www.w3.org/WAI/WCAG21/Understanding/reflow.html). In order to make that happen, content should reflow in such a way that only vertical scrolling is necessary. As currently implemented, it's not possible to visually interact with the pic2card dialog in this zoom state (see screenshot).

The above link has recommendations for how to adapt content to fit this scenario.

### Card JSON

```JSON
{}
```

### Screenshot

![image](https://user-images.githubusercontent.com/16614499/132571386-213d79bf-8298-44b6-bcd7-32de12e13613.png)
